pkg/ebpf/c/tracee.bpf.c

```diff
@@ -4361,7 +4363,7 @@ static __always_inline int tc_probe(struct __sk_buff *skb, bool ingress) {
     // See bpf_skb_event_output in net/core/filter.c.
     u64 flags = BPF_F_CURRENT_CPU;
     flags |= (u64)skb->len << 32;
```

Comment:
It depends, in the case of `net_packets` it's fine to send only the packet metadata (Src/Dst IP, port, and protocol)
but in the case of the other net events like DNS/IRC/HTTP etc.
we will need to send the whole packet because we will want to parse it in the userspace.

I can send only the  metadata in case of `net_packet` event and capture flag wasn't chosen, WYT?